server/conditions/conditions.go

```diff
@@ -43,6 +49,9 @@ func ProcessConditions(currentState *pogo.GameState, inputs []pogo.GameInputMsg,
 
 		for _, cardPriorityGroup := range cardPriorities {
 			nextState, err = applyCardsToState(cardPriorityGroup, nextState, conditions)
+			if err != nil {
+				return nextState, err
```

Comment:
I forgot to handle this error so when the conditions failed, the game just kept playing.  🤦‍♂️